.gitignore

```diff
@@ -1,2 +1,3 @@
 vendor
 phpunit.xml
+.idea
```

Comment:
This is specific to your editor - PHPStorm. It should be in your global .gitignore rather than in the project file.